package.nls.json

```
@@ -506,6 +509,12 @@
     "AWS.samcli.initWizard.launchConfigFail": "Created SAM application \"{0}\" but failed to generate launch configurations. You can generate these via {1} in the template or handler file.",
     "AWS.samcli.build.failedBuild": "\"sam build\" failed: {0}",
     "AWS.sam.localApi.request": "Sending request to local API: {0}",
+    "AWS.pickDebugHandler.noEditor": "Toolkit could not find an active editor",
+    "AWS.pickDebugHandler.invalidLanguage": "Toolkit cannot detect handlers in language: {0}",
+    "AWS.pickDebugHandler.prompt": "Create a Debug Configuration from a valid handler",
+    "AWS.pickDebugHandler.range": "Function on line {0}",
+    "AWS.pickDebugHandler.noItems": "No handlers found in current file",
+    "AWS.pickDebugHandler.noItems.detail": "Please ensure your language extension is working",
```

"Please" is noise in UI messages
```suggestion
    "AWS.pickDebugHandler.noItems.detail": "Ensure your language extension is working",
```